language/move-prover/spec-lang/src/env.rs

```diff
@@ -84,6 +88,14 @@ pub const ADDITION_OVERFLOW_UNCHECKED_PRAGMA: &str = "addition_overflow_unchecke
 /// Pragma indicating that aborts from this function shall be ignored.
 pub const ASSUME_NO_ABORT_FROM_HERE_PRAGMA: &str = "assume_no_abort_from_here";
 
+/// Internal property attached to conditions if they are injected via an apply or a module
+/// invariant.
+pub const CONDITION_INJECTED_PROP: &str = "$injected";
```

Comment:
I suppose this is how you are distinguishing between explicitly-written preconditions and postconditions and ones injected from module invariants.

Reply:
Yes